src/advanced/node-management.md

```diff
@@ -804,15 +884,192 @@ will contain the following fields:
 
 ```JSON
 {  
-   "height":13002,
-   "prev_hash":"03d19b9821a5336d3be63840ccdfa119269bd7671672a1715a71a745770f025a",
-   "proposer_id":3,
-   "state_hash":"2d5bc93ec12ab46b5197281da7557d58e438d2aac24e58c784815f4ad77a2d24",
-   "tx_count":3,
-   "tx_hash":"9135c961f53c16d791daf1d9180d34eefcd23e50c5a02f915153493d4a905fbb"
+  "height":13002,
+  "prev_hash":"03d19b9821a5336d3be63840ccdfa119269bd7671672a1715a71a745770f025a",
+  "proposer_id":3,
+  "state_hash":"2d5bc93ec12ab46b5197281da7557d58e438d2aac24e58c784815f4ad77a2d24",
+  "tx_count":3,
+  "tx_hash":"9135c961f53c16d791daf1d9180d34eefcd23e50c5a02f915153493d4a905fbb"
+}
+```
+
+### Subscribe to Transaction Commit
+
+```none
+ws://${URL}{explorer_base_path}/transactions/subscribe
+```
+
+Connects to a socket and receives notices on each new committed transaction
+starting from the moment of connection. The notices are sent to the light
+client via the socket.
+
+#### Parameters
+
+- **service_id**: integer  
+  ID of the service whose transactions are tracked.
+- **message_id**: integer  
+  ID of the tracked transaction type. `message_ID` parameter is applicable
+  only in combination with the `service_ID` parameter.
+
+#### Response
+
+Returns notifications that a new transaction has been committed to a blockchain
+block starting from the height when the client connected to the socket.
+
+Each notification is a string which can be deserialized into a JSON object that
+will contain the following fields:
+
+- **type**: string  
+  Type of the committed object. Is always equal to `"transaction"`.
+- **tx_hash**: Hash  
+  Hash of the committed transaction.
+- **service_id**: integer  
+  ID of the service where belongs the current transaction.
+- **message_id**: integer  
+  ID of the transaction type.
+- **status.type**: string  
+  Execution status kind:
+
+    - `"success"` denotes a successfully completed transaction
+    - `"error"` denotes a transaction that has returned an error (for example,
+      because of transaction parameters not satisfying context-dependent checks)
+    - `"panic"` denotes a transaction that has raised a runtime exception
+      (for example, attempted to divide by zero).
+
+- **location**: TransactionLocation  
+  Transaction position in the blockchain.
+- **proof**: ListProof  
+  Ties transaction to the root hash of the transactions Merkle tree in
+  the block.
+
+#### Response Example
+
+```JSON
+{
+  "type": "transaction",
+  "tx_hash": "fe59d6e5bbf493c4ebc112d6241a871671aa238f1c93e2726cf96fea9cb88cdd",
+  "service_id": 128,
+  "message_id": 2,
+  "status": {
+    "type": "success"
+  },
+  "location": {
+    "block_height": 317,
+    "position_in_block": 0
+  },
+  "proof": {
+    "val": "fe59d6e5bbf493c4ebc112d6241a871671aa238f1c93e2726cf96fea9cb88cdd"
+  }
+}
+```
+
+### Subscribe to Multiple Commits
```

Comment:
I don't think that `Commits` is a suitable definition here. Maybe `Events` would be better ? The point here is that we commit blocks which include transactions.